### ISSUE TITLE
Cannot see/open folders under /var/www/

### ISSUE THREAD

**Issue by o-alquimista** [flathub/io.atom.Atom#85]
Atom, when installed from Flathub, doesn't seem to have permission to access certain folders. `/var/www/`, for example, is a common place to serve files through the `httpd` web server. It doesn't show up in the directory listing when trying to "Open folder" or "Add a Project Folder".

System:
- Fedora Workstation 33 (GNOME on Wayland)
- Atom v1.51.0

**Comment by cpba** [comment]
Yes, even setting `--filesystem=host` there are still directories that are blacklisted and not available inside the sandbox, `/var` being one of them.

You can read more about this in the documentation, under filesystem access:

https://docs.flatpak.org/en/latest/sandbox-permissions.html
